draft_talon_helpers.py

```diff
@@ -5,13 +5,40 @@
 
 mod = Module()
 ctx = Context()
+ctx.matches = r"""
+title: Talon Draft
+"""
+
 mod.tag("draft_window_showing", desc="Tag set when draft window showing")
 
 draft_manager = DraftManager()
 
+
+# for rntz's context-sensitive dictation
```

Comment:
Can rntz just access the text via the draft_manager.area singleton? If something like this is needed, then I'm happy to just put it in.

Reply:
This is necessary, I think, for the same reasoning as edit.selected_text - rntz's context sensitive stuff defines these actions, and the draft UI has a better way to grab text than other apps when focused.

Reply:
It's helpful to remember that the most specific implementation of an action will be win on Talon.

I forget the exact rules atm, but e.g.
`os: windows` 

is less specific than

```
os: windows
app: talon
```

is less specific than
```
os: windows
app: talon
title: Talon Draft
```

edit.py file implements the default for `edit.selected_text()` without any ctx match, the least specific of all.